lightning/src/chain/channelmonitor.rs

```diff
@@ -2443,7 +2463,8 @@ impl<Signer: Sign> ChannelMonitorImpl<Signer> {
 		log_trace!(logger, "Processing {} matched transactions for block at height {}.", txn_matched.len(), conf_height);
 		debug_assert!(self.best_block.height() >= conf_height);
 
-		let should_broadcast = self.should_broadcast_holder_commitment_txn(logger);
+		let fulfilled_htlc_high_exposure = 5_000_000 < self.get_fulfilled_htlc_sum();
```

Comment:
Should we constify or make it a new config knob ?

I think we should give their say to node operators when it's question of risk evaluation. I pick up the same value than `max_dust_htlc_exposure`. Maybe we could pick one unified "funds at risk" value though at the same time it doesn't exactly cover the same risks...

Long-term, I think we should let node operators fine-tune their channels operations in function of their own risks analysis, instead of substituting to them (though likely too early).